docs/developer_setup.md

```diff
@@ -0,0 +1,25 @@
+# Getting Started
+
+## Cloning the Repository
+
+To begin, please clone the repository to your local machine. This will provide you with all the necessary files to run and deploy the server:
+
+```bash
+git clone https://github.com/LambdaLightSource/diamond-image-service
+```
+
+## Running Locally
+
+Navigate to the thumbor_ directory and execute server.py to start the server locally:
+
+```bash
+pip install -e .[dev]
```

Comment:
Might be worth adding in setting up a venv here.

Reply:
Added venv instructions